application/actions/threads.go

```diff
@@ -11,7 +12,18 @@ import (
 	"github.com/silinternational/wecarry-api/models"
 )
 
-// usersThreads responds to Get requests at /conversations
+// swagger:operation GET /conversations Threads Conversations
+//
+// List the User's Conversations
+//
+//
+// ---
+// responses:
+//   '200':
+//     description: A list of the user's conversations with their messages
+//     schema:
+//       items:
```

Comment:
I think you need a `//       type: array` here.

Reply:
It worked without it -- maybe since Threads is a slice.